datafusion/core/src/row/mod.rs

```diff
@@ -176,8 +176,8 @@ mod tests {
                     let batch = RecordBatch::try_new(schema.clone(), vec![Arc::new(a)])?;
                     let mut vector = vec![0; 1024];
                     let row_offsets =
```

Comment:
Would it be possible to extend these tests to include the new `WordAligned` format as well?